frontend/components/surplus/SurplusText.vue

```diff
@@ -3,43 +3,52 @@
         <TextBlock v-if="isExplanationsShown" title="What is the Maker Protocol?" class="TextBlock">
```

Comment:
Since we're reusing "What is the Maker Protocol?" text block and "What is the catch?", can you please turn them into a separate components and reuse between pages those two pages?